dev/integration_tests/windows_startup_test/lib/main.dart

```diff
@@ -38,24 +38,32 @@ void main() async {
     const MethodChannel methodChannel =
         MethodChannel('tests.flutter.dev/windows_startup_test');
 
-    // TODO(loic-sharma): Make the window invisible until after the first frame.
-    // https://github.com/flutter/flutter/issues/41980
     final bool? visible = await methodChannel.invokeMethod('isWindowVisible');
-    if (visible == null || visible == false) {
-      throw 'Window should be visible at startup';
+    if (visible == null || visible == true) {
```

Comment:
Remove `== true` to be concise 
```dart
if( visible == null || visible ) {
  throw 'Window should be hidden at startup';
}
```

If you want to be really concise.
```dart
if( visible ?? true ) {
  throw 'Window should be hidden at startup';
}
```

Reply:
It's the same for lines 49 and 53 of this file

Reply:
^^^ nits, don't worry about it.

Reply:
I considered both of these before choosing the current approach: using `??` is a little too concise for my taste, and normally I'd drop the `== true` but it looks uneven next to the `== null` check. Let me know if you feel strongly and I'll switch to the `??` approach.

Reply:
I don't mind as long as it works, which I am sure it does 🙂